### ISSUE TITLE
[Feature Request] On Linux platforms with Yama security, ask for elevated privilages

### ISSUE THREAD

**Issue by ratijas** [KDAB/GammaRay#677]
### Summary

Current status quo on Linux platforms with Yama security module enabled (which is by default) is that you either run as sudo/root, or you can't attach at all. The only workaround is to call `prctl(PR_SET_PTRACER, debugger, …)` from the process that we want to attach to, which is hardly practical. So there should be a way to elevate / ask permissions at runtime on a per-process basis without actually having user to execute any `sudo` in terminal.

Basically, some sort of PolKit integration, please.

### Versions

Operating System: Arch Linux
KDE Plasma Version: 5.24.80
KDE Frameworks Version: 5.91.0
Qt Version: 5.15.2
Kernel Version: 5.16.5-arch1-1 (64-bit)
Graphics Platform: X11
Processors: 8 × Intel® Core™ i7-6700HQ CPU @ 2.60GHz
Memory: 15.6 GiB of RAM

GammaRay: 2.11.3-1

**Comment by winterz** [comment]
a dupe of #559
